global/probe.go

```diff
@@ -19,10 +19,19 @@ package global
 
 import "time"
 
+// StatusChangeThresholdSettings is the settings for probe threshold
+type StatusChangeThresholdSettings struct {
+	// the failures threshold such as 2, 5
+	Failure int `yaml:"failure,omitempty" json:"failure,omitempty" jsonschema:"description=the failures threshold to change the status such as 3, 5,default=1"`
```

Comment:
comma in the description may result in wrong-generated documentation.

Reply:
very good finding!